source/structured-streaming.txt

```diff
@@ -160,6 +213,74 @@ more about continuous processing, see the `Spark documentation <https://spark.ap
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To use continuous processing with the MongoDB Spark Connector, 
+         add the ``trigger()`` method to the ``writeStream`` property 
+         of the streaming Dataset or DataFrame that you create from 
+         your MongoDB read stream. In your ``trigger()``, use the 
+         ``Trigger.Continuous()`` property.
+         
+         .. note:: 
+         
+            The connector populates its read stream from your MongoDB 
+            deployment's change stream. To populate your change stream, 
+            perform update operations on your database.
+         
+            To learn more about change streams, see 
+            :manual:`Change Streams </changeStreams>` in the MongoDB 
+            manual.
+         
+         Specify read stream configuration settings on your local 
+         SparkSession ``readStream``. You must specify the following 
```

Comment:
Question: Is this a Spark DataStreamReader? If so, it could be more clear to refer it to that way.
https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/DataStreamReader.html

Reply:
It is, will change throughout